Makefile

```diff
@@ -20,28 +20,33 @@ helm:
 		$(CMD)
 
 lint:
-	CMD="lint charts/cray-etcd-backup"   $(MAKE) helm
-	CMD="lint charts/cray-etcd-defrag"   $(MAKE) helm
-	CMD="lint charts/cray-etcd-base"     $(MAKE) helm
+	CMD="lint charts/cray-etcd-backup"          $(MAKE) helm
+	CMD="lint charts/cray-etcd-defrag"          $(MAKE) helm
+	CMD="lint charts/cray-etcd-base"            $(MAKE) helm
+	CMD="lint charts/cray-etcd-migration-setup" $(MAKE) helm
 
 dep-up:
-	CMD="dep up charts/cray-etcd-backup"   $(MAKE) helm
-	CMD="dep up charts/cray-etcd-defrag"   $(MAKE) helm
-	CMD="dep up charts/cray-etcd-base"     $(MAKE) helm
+	CMD="dep up charts/cray-etcd-backup"          $(MAKE) helm
+	CMD="dep up charts/cray-etcd-defrag"          $(MAKE) helm
+	CMD="dep up charts/cray-etcd-base"            $(MAKE) helm
+	CMD="dep up charts/cray-etcd-migration-setup" $(MAKE) helm
 
 test:
 	docker run --rm \
+		--user $(shell id -u):$(shell id -g) \
 		-v ${PWD}/charts:/apps \
-		${HELM_UNITTEST_IMAGE} -3 \
+		${HELM_UNITTEST_IMAGE} \
 		cray-etcd-backup \
 		cray-etcd-defrag \
-		cray-etcd-base
+		cray-etcd-base \
+		cray-etcd-migration-setup
 
 package:
 ifdef CHART_VERSIONS
-	CMD="package charts/cray-etcd-backup   --version $(word 1, $(CHART_VERSIONS)) -d packages" $(MAKE) helm
-	CMD="package charts/cray-etcd-defrag   --version $(word 2, $(CHART_VERSIONS)) -d packages" $(MAKE) helm
-	CMD="package charts/cray-etcd-base --version $(word 3, $(CHART_VERSIONS)) -d packages" $(MAKE) helm
+	CMD="package charts/cray-etcd-backup          --version $(word 1, $(CHART_VERSIONS)) -d packages" $(MAKE) helm
+	CMD="package charts/cray-etcd-defrag          --version $(word 2, $(CHART_VERSIONS)) -d packages" $(MAKE) helm
+	CMD="package charts/cray-etcd-base            --version $(word 3, $(CHART_VERSIONS)) -d packages" $(MAKE) helm
+	CMD="package charts/cray-etcd-migration-setup --version $(word 3, $(CHART_VERSIONS)) -d packages" $(MAKE) helm
```

Comment:
Is 'word 3' for both cray-defrag and cray-etcd-migration-setup  correct?

Reply:
fixed

Reply:
fixed -- not sure how it was working, maybe I would have had a publish failure.  great catch.